### ISSUE TITLE
Error r2d2 - No compatible auth mechanism

### ISSUE THREAD

**Issue by luixal** [jdrouet/catapulte#120]
Hi,

When trying out catapulte using docker, I'm getting the following error message:

```
catapulte_1  | [2021-05-04T11:35:41Z ERROR r2d2] No compatible authentication mechanism was found
```

I'm using Office365 (smtp.office365.com) as SMTP server with a valid account. It uses STARTTLS for encryption/auth.

Any idea of how to solve this?

Thanks!

**Comment by jdrouet** [comment]
Good question! I never tried with office365... Let me take a look, I'll come back to you with some doc or a fix ;)

**Comment by luixal** [comment]
Hi,

I just had a look at the code and just found this in [smtp.rs#L170](https://github.com/jdrouet/catapulte/blob/ee4696801a86ecaa6ce90bc15f4746fe5582f0e9/src/service/smtp.rs#L170):

```rust
let _tls_enabled = env_test_util::TempEnvVar::new("SMTP_TLS_ENABLED").with("false");
```
Once I hadded the env var ```SMTP_TLS_ENABLED="true"``` everything worked like a charm.

I though I had missed it in the docs so I reviewed but didn't see any mention to it. I've just made a couple of PRs adding it to the wiki and the readme example.

Thanks for this project, really useful!
